druid-shell/src/common_util.rs

```diff
@@ -152,3 +155,8 @@ impl Default for ClickCounter {
         ClickCounter::new(MULTI_CLICK_INTERVAL, MULTI_CLICK_MAX_DISTANCE)
     }
 }
+
+#[cfg(not(target_arch = "wasm32"))]
```

Comment:
why not wasm32?

Reply:
its unused. and allowing unused was giving me issues.

Reply:
fixed up. had to do some rust reading around the error i was seeing last night. =)